coreapps-web-2.2/pom.xml

```diff
@@ -0,0 +1,147 @@
+  <modelVersion>4.0.0</modelVersion>
+      <parent>
+        <groupId>org.openmrs.module</groupId>
+        <artifactId>coreapps</artifactId>
+        <version>1.32.0-SNAPSHOT</version>
+     </parent>
+     <artifactId>coreapps-web-2.2</artifactId>
+     <packaging>jar</packaging>
+     <name>coreapps Module web 2.2</name>
+     <description>2.2 web project for coreapps</description>
+     
+     <properties>
+        <openMRSVersion>2.5.0-SNAPSHOT</openMRSVersion>
+        <reportingVersion>0.10.3</reportingVersion>
+        <calculationVersion>1.3.0-SNAPSHOT</calculationVersion>
+     </properties>
+     
+     <dependencies>
+         <dependency>
+           <groupId>${project.parent.groupId}</groupId>
+           <artifactId>${project.parent.artifactId}-api</artifactId>
+           <version>${project.parent.version}</version>
+           <scope>provided</scope>
+         </dependency>
+         
+          <dependency>
+            <groupId>org.openmrs.api</groupId>
+            <artifactId>openmrs-api</artifactId>
+            <version>${openMRSVersion}</version>
+            <scope>provided</scope>
+        </dependency>
+        
+        <dependency>
+            <groupId>org.openmrs.api</groupId>
+            <artifactId>openmrs-api</artifactId>
+            <type>test-jar</type>            
+            <version>${openMRSVersion}</version>
+            <scope>test</scope>
+        </dependency>
+        
+        <dependency>
+            <groupId>org.openmrs.test</groupId>
+            <artifactId>openmrs-test</artifactId>
+            <version>${openMRSVersion}</version>
+            <type>pom</type>
+            <scope>test</scope>
+        </dependency>
+        
+        <dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>emrapi-api</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>reporting-api</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>calculation-api</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>metadatamapping-api</artifactId>
+		</dependency>
+		                         <dependency>
+				<groupId>org.openmrs.module</groupId>
+				<artifactId>serialization.xstream-api-1.9</artifactId>
+				<scope>test</scope>
+			</dependency>
+		
+			<dependency>
+				<groupId>org.openmrs.module</groupId>
+				<artifactId>serialization.xstream-api-1.10</artifactId>
+				<scope>test</scope>
+			</dependency>
+		
+			<dependency>
+				<groupId>org.openmrs.module</groupId>
+				<artifactId>serialization.xstream-api-2.0</artifactId>
+				<scope>test</scope>
+			</dependency>
+			
+			<dependency>
+				<groupId>org.openmrs.module</groupId>
+				<artifactId>serialization.xstream-api</artifactId>
+				<scope>test</scope>
+			</dependency>
+
+                        <dependency>
```

Comment:
```suggestion
		<dependency>
```

Reply:
@dkayiwa @mozzy11 @mks-d  @ibacher so on updating my pom as suggested by Moses, i do run into this error https://pastebin.com/Dvv6J628